pkg/webservice/tests/handlers_test.go

```diff
@@ -16,20 +16,37 @@
  limitations under the License.
 */
 
-package webservice
+package tests
 
 import (
 	"encoding/json"
+	"github.com/apache/incubator-yunikorn-core/pkg/common/resources"
```

Comment:
The import orders are not good. Could you please run `make lint` locally to see these warnings before submitting the patch?